docs/ism/policies.md

```diff
@@ -321,6 +322,26 @@ Parameter | Description | Type | Required | Default
 }
 ```
 
+### index_priority
+
+Set the priority for the index in a specific state. Unallocated shards of indices are recovered in the order of their priority, whenever possible.
+
+The `index_priority` operation has the following parameter:
+
+Parameter | Description | Type | Required | Default
+:--- | :--- |:--- |:--- |
```

Comment:
it seems we miss another `:---` here, the table is not rendered out correctly, and please check there are other not rendered table in this `policies.md` page

Reply:
changed.